ui/webui/resources/BUILD.gn

```diff
@@ -391,6 +391,7 @@ leo_icons = [
   "trash.svg",
   "tune.svg",
   "uphold-color.svg",
+  "upload.svg",
```

Comment:
I think this file is missing from this commit unless it already exists in the repo

Reply:
These are files from node_modules @bridiver